spec/exception_spec.rb

```diff
@@ -75,6 +75,9 @@
       expect(Azure::Armrest::BadGatewayException).to_not be_nil
       expect(Azure::Armrest::GatewayTimeoutException).to_not be_nil
       expect(Azure::Armrest::TooManyRequestsException).to_not be_nil
+      expect(Azure::Armrest::PayloadTooLargeException).to_not be_nil
+      expect(Azure::Armrest::ServiceUnavailableException).to_not be_nil
+      expect(Azure::Armrest::LoopDetectedException).to_not be_nil
     end
```

Comment:
@bronaghs I suppose I could add something, though the mere existence of those classes tells us that the generation happened successfully at least.